recipes/mlgenotype/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "mlgenotype" %}
+{% set version = "0.1.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  #url: "https://test-files.pythonhosted.org/packages/40/e4/483d6d723f3727abbd96404009b773408cebe8d6a0c6478a6f43e9f4d88f/mlgenotype-0.1.12.tar.gz"
+  sha256: e89eb045be093204671c6110a341b4b1dd8ae78c41ba1b1e59b3b4a4c8cabc76
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
add `--ignore-installed --no-deps` (since you have listed deps in `run` section)